### PR TITLE
[warnings] fixed reserved-identifer warnings

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: server/proxy/pf_context.c [hunk 1]
```diff
@@ -89,6 +93,12 @@ void StaticChannelContext_free(pServerStaticChannelContext* ctx)
 	free(ctx);
 }
 
+static void HashStaticChannelContext_free(void* ptr)
+{
+	pServerStaticChannelContext* ctx = (pServerStaticChannelContext*)ptr;
```
Comment (pnowack) [inline server/proxy/pf_context.c:98]: Casting void pointers is a superfluous action

```suggestion
	pServerStaticChannelContext* ctx = ptr;
```

File: server/proxy/pf_context.c [hunk 2]
```diff
@@ -129,7 +139,7 @@ static BOOL client_to_proxy_context_new(freerdp_peer* client, rdpContext* ctx)
 	obj->fnObjectEquals = ChannelId_Compare;
 
 	obj = HashTable_ValueObject(context->channelsByFrontId);
-	obj->fnObjectFree = StaticChannelContext_free;
+	obj->fnObjectFree = HashStaticChannelContext_free;
```
Comment (pnowack) [inline server/proxy/pf_context.c:142]: Nit: The name now reads to me as "hash of (a) static channel context", but this doesn't seem to be the case here. Maybe there is a different way to name this (I am actually fine with the previous `StaticChannelContext_free` (or the same written in snake case))?

**Review by pnowack — APPROVED** [review]
Just two nits. Aside from that, LGTM

File: server/proxy/pf_context.c (hunk 1 above)
Comment (akallabeth) [reply server/proxy/pf_context.c:98]: thx for the hint, will remember that for the next pr ;)

File: server/proxy/pf_context.c (hunk 2 above)
Comment (akallabeth) [reply server/proxy/pf_context.c:142]: hard to name it, as it only exists due to `C` lack of overloaded parameter support.

**Pull request merged**